lib/excon/utils.rb

```diff
@@ -74,13 +74,16 @@ def parse_query_string(string)
       params = {}
 
       string.split(/[&;] */n).each do |pair|
-        key, value = pair.split('=', 2).map { |x| CGI.unescape(x) }
+        key, value = pair.split('=', 2).map do |x|
+          # unescape, based on URI::Addressable unencode
```

Comment:
Extremely minor thing, but just something I noticed.

```suggestion
          # unescape, based on Addressable::URI unencode
```

Reply:
:facepalm: good catch. It's minor now, but I imagine it could save some major headaches in the future when we've all forgotten the original context/intent. Thanks!